dashboard/config/scripts/ECSPD3-Unit2.script

```diff
@@ -33,7 +33,7 @@ level 'ECSPD U2 D 4-6'
 level 'ECSPD U2 D 7-9 Intro'
 level 'ECSPD U2 D 10-12 Intro'
 level 'ECSPD Binary Odometer'
-level 'ECSPD Why Binary - The Internet Video'
+level 'U1L5 How the Internet Works - Video'
```

Comment:
Was this intentional? It would appear that the "Why Binary - The Internet" video has been completely removed (I'm seeing ten removed `.external` levels and nine added `StandaloneVideo` levels)